editor/src/components/canvas/canvas-strategies/strategies/ancestor-metastrategy.tsx

```diff
@@ -57,7 +57,11 @@ export function ancestorMetaStrategy(
     const unrolledChild = unrolledChildren[0]
 
     // Is the selected element an only child?
-    const siblings = MetadataUtils.getSiblingsUnordered(canvasState.startingMetadata, unrolledChild)
+    const siblings = MetadataUtils.getSiblingsUnordered(
```

Comment:
At this point is it not just using `getSiblingsOrdered`? In which case we can bin this off too

Reply:
Oooh, yes, I was meaning to look back at this after creating the PR.

Reply:
Fixed in d1565dc9d.